From ad5a5ee67946d4d626b98222dd90e0817738a114 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 14 Aug 2013 13:46:10 +0800
Subject: [PATCH 232/304] ENGR00267092 BT: enable BLE feature

Enable BLE feature. Test it using:
1.AR3002 BT + sabreSD
2.Bluetooth smart device supporting BLE: Polar H7 Heart Rate Sensor.
3.Test apk:BluetoothLeGatt.apk

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx5x/imx53_smd.mk     | 3 ++-
 imx6/arm2_6dq.mk       | 3 ++-
 imx6/arm2_6sl.mk       | 3 ++-
 imx6/evk_6sl.mk        | 3 ++-
 imx6/hdmidongle_6dq.mk | 3 ++-
 imx6/sabreauto_6q.mk   | 3 ++-
 imx6/sabresd_6dq.mk    | 3 ++-
 7 files changed, 14 insertions(+), 7 deletions(-)

diff --git a/imx5x/imx53_smd.mk b/imx5x/imx53_smd.mk
index ec1efe6..dcae2a1 100644
--- a/imx5x/imx53_smd.mk
+++ b/imx5x/imx53_smd.mk
@@ -26,7 +26,8 @@ PRODUCT_COPY_FILES += \
     frameworks/native/data/etc/android.hardware.camera.front.xml:system/etc/permissions/android.hardware.camera.front.xml \
     frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
     frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
-    frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml
+    frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
+    frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # audio related files
 PRODUCT_COPY_FILES += \
diff --git a/imx6/arm2_6dq.mk b/imx6/arm2_6dq.mk
index ffc3cf6..d4d03cb 100644
--- a/imx6/arm2_6dq.mk
+++ b/imx6/arm2_6dq.mk
@@ -29,7 +29,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # for PDK build, include only when the dir exists
 # too early to use $(TARGET_BUILD_PDK)
diff --git a/imx6/arm2_6sl.mk b/imx6/arm2_6sl.mk
index 08f75b4..e8fbaec 100644
--- a/imx6/arm2_6sl.mk
+++ b/imx6/arm2_6sl.mk
@@ -28,4 +28,5 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
diff --git a/imx6/evk_6sl.mk b/imx6/evk_6sl.mk
index dae9c6a..d517308 100644
--- a/imx6/evk_6sl.mk
+++ b/imx6/evk_6sl.mk
@@ -35,7 +35,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # for PDK build, include only when the dir exists
 # too early to use $(TARGET_BUILD_PDK)
diff --git a/imx6/hdmidongle_6dq.mk b/imx6/hdmidongle_6dq.mk
index 53523c0..a7ddc68 100644
--- a/imx6/hdmidongle_6dq.mk
+++ b/imx6/hdmidongle_6dq.mk
@@ -35,7 +35,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # for PDK build, include only when the dir exists
 # too early to use $(TARGET_BUILD_PDK)
diff --git a/imx6/sabreauto_6q.mk b/imx6/sabreauto_6q.mk
index 0c04a47..e836c3a 100644
--- a/imx6/sabreauto_6q.mk
+++ b/imx6/sabreauto_6q.mk
@@ -34,7 +34,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # for PDK build, include only when the dir exists
 # too early to use $(TARGET_BUILD_PDK)
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index 028acfc..1629654 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -36,7 +36,8 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.touchscreen.multitouch.jazzhand.xml:system/etc/permissions/android.hardware.touchscreen.multitouch.jazzhand.xml \
 	frameworks/native/data/etc/android.software.sip.voip.xml:system/etc/permissions/android.software.sip.voip.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
-	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml
+	frameworks/native/data/etc/android.hardware.usb.accessory.xml:system/etc/permissions/android.hardware.usb.accessory.xml \
+	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml
 
 # for PDK build, include only when the dir exists
 # too early to use $(TARGET_BUILD_PDK)
-- 
1.8.0

